plugins/notifications-backend-module-email-notification-processor/README.md

```diff
@@ -0,0 +1,23 @@
+# @backstage/plugin-notifications-backend-module-email-notification-processor
+
+The email-notification-processor backend module for the notifications plugin.
+The idea is to send the notification details via email when the notification severity is above the configured threshold.
+
+Example configuration:
+
+```yaml
+backend:
+  email-notification-processor:
+    enabled: boolean # mandatory
+    severityThreshold: 3 # mandatory. 1-4.
+    sender: superstar@gmail.com # mandatory.
+    recipient: theadmin@gmail.com # mandatory
+    hostname: smtp.gmail.com # mandatory
```

Comment:
I think this should be separated to `transport` config and you could support also other transports than smtp. Maybe use something like nodemailer for this:https://www.nodemailer.com/transports/

Reply:
we need only smtp at the moment